observability/tools/readme.md

```diff
@@ -3,11 +3,12 @@
 There are a number of modern tools to make systems observable. While identifying and/or creating tools that work for your system, here are a few things to consider to help guide the choices.
 
 - Must be simple to integrate and easy to use.
-- It must be possible to aggregrate and visualize data.
+- It must be possible to aggregate and visualize data.
 - Tools must provide real-time data.
-- Must be able to guide users to the problem area with suitable, adequete end-to-end context.
+- Must be able to guide users to the problem area with suitable, adequate end-to-end context.
 
 ## Choices
 
 - [Loki](./loki.md)
```

Comment:
I'm a bit confused. I think we should clearly separate metrics from logging (Loki belongs to the latter, together with Splunk and ELK stack). Technically speaking, observability is a property of a system, while logging&monitoring are defined practices and tools. I would also clearly separate infrastructure monitoring vs application monitoring (including APM systems like AppInsights in the latter). But I think overall this PR can be merged as it's a great starting point.